config/crd/v1/bases/velero.io_backuprepositories.yaml

```diff
@@ -53,7 +53,7 @@ spec:
               repositoryType:
                 description: RepositoryType indicates the type of the backend repository
                 enum:
-                - kopia
+                - unified
```

Comment:
Where do you define the association between kopia and unified?  
From the perspective of code path, it's the unified repo, but from the API point of view, we have to provide a way for user to explicitly tell velero the type of the backup repository right?

Reply:
As the discussion, we will have the `repositoryType` reflect the final repo provider, so here we keep it as `kopia`.

In future, when we have more than one repo providers managed by Unified Repo, we will add this `repositoryType` to BSL, so that Unified Repo could choose the repo provider according to this field.
For now, since we have only one repo provider, for simplification, we don't have it in BSL, so we just make a hardcode of the `repositoryType`:

- For PVB/PVR, for Kopia path, we always set it to `kopia`
- For other data movements, we always set it to `kopia`